<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1336</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="clnt_tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="cred_clnt_tab">
       <attribute name="title">
        <string>Персональные данные</string>
       </attribute>
       <widget class="QLabel" name="name_clnt_label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="email_clnt_label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="phone_clnt_label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Номер телефона</string>
        </property>
       </widget>
       <widget class="QLabel" name="bd_clnt_label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>220</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Дата рождения</string>
        </property>
       </widget>
       <widget class="QLabel" name="name_clnt_value">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>110</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="email_clnt_value">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>150</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="phone_clnt_value">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>180</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="bd_clnt_value">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>220</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="change_password_clnt_button">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>330</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить пароль</string>
        </property>
       </widget>
       <widget class="QLabel" name="change_password_clnt_label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>340</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Введите новый пароль</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="change_password_clnt_value">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>340</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="save_password_clnt_button">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>340</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_cred_clnt_button">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>380</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Редактировать данные</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deactivate_account_button">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>420</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Деактивировать аккаунт</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="books_clnt_tab">
       <attribute name="title">
        <string>Книги</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="books_table_widget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Автор</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Жанр</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Количество</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Год публикации</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Цена</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Аннотация</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item row="4" column="1">
            <widget class="QDateEdit" name="min_date_value"/>
           </item>
           <item row="5" column="1">
            <widget class="QDateEdit" name="max_date_value"/>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="find_by_year_button">
             <property name="text">
              <string>Найти по указанному времени</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QDoubleSpinBox" name="min_price_value">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QDoubleSpinBox" name="max_price_value">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QPushButton" name="find_by_price">
             <property name="text">
              <string>Найти по указанной цене</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Жанр</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QComboBox" name="genre_combo_box">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Автор</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QComboBox" name="authors_combo_box"/>
           </item>
           <item row="12" column="1">
            <widget class="QPushButton" name="sort_by_pop_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Отсортировать по популярности</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QPushButton" name="clear_filters_button">
             <property name="text">
              <string>Очистить фильтры</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="orders_clnt_tab">
       <attribute name="title">
        <string>Заказы</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="create_order_button">
             <property name="text">
              <string>Оформить заказ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="processed_orders_button">
             <property name="text">
              <string>Исполняемые заказы</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="all_orders_button">
             <property name="text">
              <string>Все заказы</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="orders_table_widget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Номер заказа</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Стоимость</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата оформления</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Текущий статус</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1336</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
